skunkworks_crow/src/main/java/org/odk/share/views/ui/bluetooth/BtReceiverActivity.java

```diff
@@ -210,9 +238,6 @@ private Disposable addDownloadEventSubscription() {
                 .observeOn(schedulerProvider.androidThread())
                 .subscribe(downloadEvent -> {
                     switch (downloadEvent.getStatus()) {
-                        case QUEUED:
```

Comment:
Why this is removed?

Reply:
I think the download job was queued is no need to let our user know, all they need to know is is transferring started and finished, what do you think? @lakshyagupta21

Reply:
Few events are added for the understanding of developer, that's the only reason of adding it here, you can remove the toast message instead adding some log over there would be great

Reply:
Great, will replace that using a timber log.